docker-compose.yml

```diff
@@ -1,8 +1,28 @@
 version: "3.9"
 services:
-  wordpress:
-    # Here you put configuration of continer for wordpress that would use image that you build. Ports of webserver should be exposed to lacalhost
-    #...
   database:
-    # Here you put configuration of continer for MySQL that would use any mysql image you like
-    #...
+    image: mysql:8
+    ports:
+    - "3306:3306"
+    restart: unless-stopped
+    env_file: .env
+    volumes:
+      - dbdata:/var/lib/mysql
+    command: '--default-authentication-plugin=mysql_native_password'
+  
+
+  wordpress:
+    build: .
+    #image: danpaldev/wp-pluginv9:2179fd0
+    ports:
+      - "80:80"
+    restart: always
+    volumes:
+      - wordpress:/var/www/app/wordpress
```

Comment:
Great that you use volumes here. Question, why do you save whole wordpress directory?

Reply:
In order to achieve persistance of user uploads, themes and plugins, you need to preserve the database info and the ```wordpress/wp-content``` directory. So yes... there's no need to persist the whole core directory.